Tests/HttpPipelineTests/__Snapshots__/HttpPipelineTests/testBasicAuth_Authorized.1.Conn.txt

```diff
@@ -13,10 +13,8 @@ Body: (Data, 0 bytes)
 
 Response:
 ---------
-Status 200 (ok)
-Headers: [
-  Content-Type: text/html; charset=utf8
-]
-Bytes: 19
+Status 200 OK
```

Comment:
Having a little fun with the response formatting to make it look more like a raw HTTP response.